app/lib/activitypub/tag_manager.rb

```diff
@@ -96,12 +96,14 @@ def uri_to_resource(uri, klass)
       when 'Account'
         klass.find_local(uri_to_local_id(uri, :username))
       else
-        klass.find_by(id: uri_to_local_id(uri))
+        StatusFinder.new(uri).status
```

Comment:
i'm not sure this is the correct behavior here. At the very least, it should be explicitly checking for `when 'Status'` before using StatusFinder

Reply:
@nightpool To be fair we don't parse any URIs which are not Account or Status.